api/services/elastigroup/azure/spotVms/paths/vmSignal.yaml

```diff
@@ -1,7 +1,7 @@
 post:
   summary: "Create VM Signal"
   description:
-    CThe VM signal API is used for notifying Spot about the VM state so that Spot can act accordingly.
```

Comment:
@vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master